docs/products/opensearch/howto/opensearch_alerting.rst

```diff
@@ -0,0 +1,120 @@
+
+   In the following example, we are using a ``sample-host-health`` index as datasource to create a simple alert to check cpu load, action will be triggered when average of ``cpu_usage_percentage`` over ``3`` minutes is above ``75%``
+
+To create the monitor programmatically, you can skip the following manual steps to :ref:`here<Code>`
+
+To define an alert:
+
+1. Log in to the `Aiven web console <https://console.aiven.io>`_ and select your OpenSearch service.
+
+2. Click the **Overview** tab -> **OpenSearch Dashboards** under `Connection Information`
+
+   This opens the OpenSearch Dashboard
+
+3. In OpenSearch Dashboard Click  on top left to open the left side panel -> Under `OpenSearch Plugins` -> **Alerting**
+
+.. note::
+    To configure each alert there are ``Monitor``, ``Data source``, ``Query``, ``Trigger`` and ``Destination`` needs to be created.
+
+4. Click the **Destination** tab -> **Add destination**
+   
+   **Name** -> ``slack-test``
+
+   **Type** -> ``Slack``
+
+   **Settings** **Webhook URL:** -> ``https://your_slack_webhook_URL``
+.. note::
+   Destination Type can be: `Amazon Chime`, `Slack`, `Custom webhook` or `Email`
+
+.. important::
+   When using email you need to have a SMTP server configured for a valid domain to deliver email notifications
+
+5. Click the **Monitors** tab -> **Create monitor**
+
+6. **Monitor details**
+   
+   **Monitor name** -> ``High CPU Monitor``
+
+   **Monitor name** -> ``Per query monitor``
+   
+   **Monitor defining method** -> ``Visual editor`` 
+
+   **Schedule** **Frequency** -> ``By interval``
+
+   **Run every** -> ``1`` ``Minutes``
+
+.. note::
+   Schedule Frequency can be `By internal`, `Daily`, `Weekly`, `Monthly`, `Custom CRON expression`
+
+
+7. **Data source** 
+ 
+   **index** -> ``sample-host-health``
+
+   **Time field** -> ``timestamp``
+
+8. **Query**
+
+   **Metrics** -> **Add metric** 
+
+   **Aggregation** ``avrerage()`` **Field** ``cpu_usage_percentage``
+
+   **Time range for the last** ``3`` ``minutes``
+
+9. **Add trigger**
+
+   **Trigger name** -> ``high_cpu``
+
+   **Severity level** -> ``1 (Highest)``
+
+   **Trigger condition** ``IS ABOVE`` ``75``
+
+.. note::
+   You can see a visual graph below trigger with the index data and the trigger condition you have defined as a red line
+
+10. **Actions**
+     
+   **Action name** -> ``slack``
+
+   **Destination** -> ``slack-test``
+
+   **Message subject** -> ``High CPU Test Alert``
+
+.. note::
+   Multiple Actions can be defined, in this example we will define one action to send notification to destination we have defined in step 4
+
+   **Message** can be adjusted as needed, check **Message Preview** to see the sample and use **Send test message** to validate notification delivery
+
+1.  Click on **Create** and your monitor is ready!
+
+.. _Code:
+
+Code
+====
+
+The following code is for creating this monitor programmatically.
```

Comment:
Can we add some generic instructions on how to do this? Example is perfect, but extra lines on what expected to be in the JSON, specific endpoint ("_plugins/_alerting/monitors"). It would help the reader if those things are pointed out.